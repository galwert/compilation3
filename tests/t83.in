void main()
{
    return;
}

int get1()
{
    return 1;
}

void main()
{
    return;
}