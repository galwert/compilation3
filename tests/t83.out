---end scope---
---end scope---
line 11: identifier main is already defined
